src/model.ts

```diff
@@ -271,10 +271,14 @@ export class Model extends EventBus<any> implements CommandDispatcher {
     return [...this.coreHandlers, ...this.allUIPlugins, this.history];
   }
 
-  get coreHandlers(): CommandHandler<Command>[] {
+  get coreHandlers(): CommandHandler<CoreCommand>[] {
```

Comment:
you are conflicting with yourself :smile: https://github.com/odoo/o-spreadsheet/commit/653874056a95081eba1d61236c5c6fad2c8c5f42

Reply:
Forgot to rebase 😅